src/styles/utils/_mixins.scss

```diff
@@ -0,0 +1,19 @@
+@mixin stopwatch-size {
+  width: $stopwatch-size;
+  height: $stopwatch-size;
+}
+
+@mixin min-hand-size {
+  height: $hand-min-height;
+  width: $hand-min-width;
+}
+
+@mixin sec-hand-size {
+  height: $hand-sec-height;
+  width: $hand-sec-width;
+}
+
+@mixin center-size {
+  width: $center-size;
+  height: $center-size;
+}
```

Comment:
Strange solution 🙂 These are squares, and logic is same for every of it. Just create one and then pass needed value to it

Reply:
Also you have few circles on page, so will be useful to create mixin for circle too (you may reuse square for it and add border-radius)

Reply:
Done!